apps/pom.xml

```
@@ -128,9 +128,10 @@
 		<!-- The default DB that will be used in integration tests. -->
 		<!-- Note: See gov.cms.bfd.model.rif.schema.DatabaseTestHelper for details
 			on the default 'jdbc:bfd-test:' URL here. -->
-		<its.db.url>jdbc:bfd-test:hsqldb:mem</its.db.url>
+		<its.db.url>jdbc:bfd-test:tc</its.db.url>
 		<its.db.username/>
 		<its.db.password/>
+		<its.testcontainer.db.image>postgres:14.6</its.testcontainer.db.image>
```

This is a little speedier with a smaller image...
```suggestion
		<its.testcontainer.db.image>postgres:14.6-alpine</its.testcontainer.db.image>
```